src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -34,7 +37,7 @@ def update
           user: User.session
         }
       end
-      send_notifications_information_rabbitmq(@read_count, @unread_count)
+      send_notifications_information_rabbitmq(!delivered, @count)
```

Comment:
Later when #16101 is merged and we can select both read and unread notifications, this line won't send both counts to Grafana...

Reply:
So, we will be losing that feature...

Reply:
> The feature of doing the opposite action than what the button said it would do, yes

I don't understand... The code we had before could toggle both read and unread notifications, now that we still can't see both read and unread notifications, that's no big deal, but once the PR I posted before is merged, we can see them both and then we could toggle several notifications at once.

With this change, we're going to prevent it